extensions/wikia/PhalanxII/fallback/Phalanx.class.php

```diff
@@ -72,9 +72,9 @@ static public function getTypeNames( $typemask ) {
 
 		$types = array();
 
-		//iterate for each module for which block is saved
-		for ($bit = $typemask&1, $type=1; $typemask; $typemask>>=1, $bit = $typemask&1, $type<<=1) {
-			if (!$bit) continue; //skip not used modules
+		// iterate for each module for which block is saved
+		for ( $bit = $typemask&1, $type = 1; $typemask; $typemask >>= 1, $bit = $typemask&1, $type <<= 1 ) {
+			if ( !$bit ) continue; // skip not used modules
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add curly braces around the nested statement(s). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) 2 statements were found on this line. Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)